charts/argo-controller/templates/config.yaml

```diff
@@ -49,5 +49,10 @@ data:
       secondsAfterSuccess: {{ .workflowTTL.secondsAfterSuccess }}
       secondsAfterFailure: {{ .workflowTTL.secondsAfterFailure }}
     {{- end }}
+  metricsConfig: |
+    disableLegacy: false 
```

Comment:
I saw one of the Argo maintainers say that these should be disabled for new versions, can we set this to `true`?

Reply:
Yeah! Sounds good.